pkg/pool-linear/contracts/LinearPool.sol

```diff
@@ -150,6 +151,31 @@ abstract contract LinearPool is ILinearPool, IGeneralPool, IRateProvider, BasePo
         _setTargets(mainToken, lowerTarget, upperTarget);
     }
 
+    /**
+     * @dev Inserts a zero-valued entry in the `assetManagers` array at the BPT token index, making BPT not be managed
+     * even if the main or wrapped token are.
+     */
+    function _insertNullBptAssetManager(
+        IERC20 mainToken,
+        IERC20 wrappedToken,
+        address[] memory assetManagers
+    ) private view returns (address[] memory) {
+        (, , uint256 bptIndex) = _getSortedTokenIndexes(mainToken, wrappedToken, this);
+
+        address[] memory extendedAssetManagers = new address[](assetManagers.length + 1);
+        for (uint256 i = 0; i < extendedAssetManagers.length; ++i) {
+            if (i < bptIndex) {
+                extendedAssetManagers[i] = assetManagers[i];
+            } else if (i > bptIndex) {
+                extendedAssetManagers[i] = assetManagers[i - 1];
+            } else {
+                extendedAssetManagers[i] = address(0);
+            }
```

Comment:
```suggestion
            }
```

optimizoor

Reply:
:joy: given this happens only once in the constructor, I'd rather be extra explicit